File: CS50 Problem Sets/Problem Set 6/sentimental/mario.py
from cs50 import get_int

# Get height
# Then check height
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# For loop row
for i in range(0, height, 1):
    # for loop column
    for j in range(0, height, 1):
        if (i + j < height - 1):
            print(" ", end="")
        else:
            # print hash
            print("#", end="")
    print()

File: CS50 Problem Sets/Problem Set 1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Enter value: ");
    }
    while(height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int k = 0; k < height - i - 1; k++){
            printf(" ");
        }


        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }



        printf("  ");



        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}